docs/src/developer-guide/architecture/index.md

```diff
@@ -5,7 +5,7 @@ edit_link: https://github.com/eslint/eslint/edit/main/docs/src/developer-guide/a
 
 ---
 
-<center><img alt="dependency graph" src="dependency.svg"></center>
+<center><img alt="dependency graph" src="./../../assets/images/dependency.svg"></center>
```

Comment:
It would be nice if we could implement aliases by some how so that we can use `images/dependency.svg` instead of adding the complete links